repair/repair.cc

```diff
@@ -1021,13 +1021,13 @@ future<int> repair_service::do_repair_start(sstring keyspace, std::unordered_map
     auto& erm = germs->get();
     auto& topology = erm.get_token_metadata().get_topology();
 
+    if (erm.get_replication_strategy().get_type() == locator::replication_strategy_type::local) {
+        rlogger.info("Nothing to repair for keyspace {} with local replication strategy", keyspace);
+        co_return 0;
+    }
+
```

Comment:
The gain of the shortcut is low. But it make the api less consistent and introduce more special cases.  This might surprise user of the api (e.g., rest api user) which expect the id is starting from 1.

It makes more sense to just return a normal repair id and let the logs like `repair[{}]: starting user-requested repair for keyspace` still show as a normal repair. 

We can do the shortcut later and log something similar to this:

```
    if (cfs.empty()) {
        rlogger.info("repair[{}]: completed successfully: no tables to repair", id.uuid());
        co_return id.id;
    }
```